_topic_map.yml

```diff
@@ -883,7 +883,7 @@ Topics:
   - Name: Working with nodes
     File: nodes-nodes-working
   - Name: Managing Nodes
-    File: nodes-nodes-managing
+                                  File: nodes-nodes-managing
```

Comment:
You have some extra spacing here that could be causing the build to fail

Reply:
This is most likely the cause of the build error.

Reply:
Removed the extra spaces preceding File: nodes-nodes-managing.